src/nbl/video/device_capabilities/device_limits.json

```diff
@@ -903,81 +903,77 @@
-                    "value": true,
-                    "expose": "DISABLE"
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderSignedZeroInfNanPreserveFloat64",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormPreserveFloat16",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormPreserveFloat32",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormPreserveFloat64",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormFlushToZeroFloat16",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormFlushToZeroFloat32",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderDenormFlushToZeroFloat64",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTEFloat16",
-                    "value": false,
-                    "comment": ["ROADMAP2024 but no good support yet"]
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTEFloat32",
-                    "value": false,
-                    "comment": ["ROADMAP2024 but no good support yet"]
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTEFloat64",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTZFloat16",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTZFloat32",
-                    "value": false
+                    "value": true
                 },
                 {
                     "type": "bool",
                     "name": "shaderRoundingModeRTZFloat64",
-                    "value": false
+                    "value": true
```

Comment:
this is a mess and needs to get fixed

Reply:
https://github.com/Devsh-Graphics-Programming/Nabla/pull/829#discussion_r1944463467

Reply:
you can only DISABLE and `value=true` the 32bit ones, rest you need to DISABLE and `value=false`

Reply:
sorry if its `true` then it needs to be `REQUIRE` my bad